templates/CRM/Admin/Page/Options.tpl

```diff
@@ -58,7 +58,7 @@
 
 <div class="crm-content-block crm-block">
 {if $rows}
-{if $isLocked ne 1}
+{if !empty($isLocked) and $isLocked ne 1}
```

Comment:
I'm still struggling with this one - should it be

if (!isset($isLocked) || $iiLocked ne 1)

ie NULL or zero should be the same - & should be TRUE

NULL would be true on the first condition & 0 on the second  so both would be TRUE

1 would be false on the first AND false on the second

Reply:
ah I fixed it in the other place you found

Reply:
Updated to be like what we have done in L159 now